### PR TITLE
Use Bootstrap class for table

### PR DESCRIPTION
**Pull request by nicholaschuayunzhi**
Add bootstrap class for for striped and borded tables

Related https://github.com/MarkBind/markbind-cli/pull/33

### CODE REVIEW

File: lib/markdown-it/index.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ markdownIt.normalizeLink = require('./normalizeLink');
 
 // fix table style
 markdownIt.renderer.rules.table_open = (tokens, idx) => {
-  return '<table class="markbind-table table">';
+  return '<table class="markbind-table table-bordered table-striped table">';
```
Comment (acjh) [inline lib/markdown-it/index.js]: Why did `table` become the last class?

Comment (nicholaschuayunzhi) [reply lib/markdown-it/index.js]: My bad. Should it be ` markbind-table table table-bordered table-striped` instead? Based on alphabetical order.

Comment (acjh) [reply lib/markdown-it/index.js]: Yes. Sometimes special classes like `markbind-table` can be excluded from the "alphabetical order" convention.

**Comment by acjh** [comment]
Please squash the commits.

**Comment by nicholaschuayunzhi** [comment]
Squashed! @acjh

**Pull request merged**